src/platform/packages/shared/presentation/presentation_util/src/open_lazy_flyout.tsx

```
@@ -74,27 +77,26 @@ export const openLazyFlyout = (params: OpenLazyFlyoutParams) => {
     onClose();
   });
 
-  const flyoutRef = core.overlays.openFlyout(
-    toMountPoint(
-      <LazyFlyout
-        closeFlyout={onClose}
-        loadContent={loadContent}
-        core={core}
-        ariaLabelledBy={ariaLabelledBy}
-      />,
-      core
-    ),
+  const flyoutRef = core.overlays.openSystemFlyout(
+    <LazyFlyout
+      closeFlyout={onClose}
+      loadContent={loadContent}
+      core={core}
+      ariaLabelledBy={ariaLabelledBy}
+    />,
     {
+      session: 'start',
       size: 500,
       type: 'push',
       paddingSize: 'm',
       maxWidth: 800,
+      resizable: true,
       ownFocus: true,
-      isResizable: true,
       outsideClickCloses: true,
       className: 'kbnPresentationLazyFlyout',
       'aria-labelledby': ariaLabelledBy,
       onClose,
+      title: 'title placeholder',
```

The new flyout service requires passing a title prop.
For now I added a placeholder value so the flyout renders correctly and avoids runtime errors.

According to the documentation, parent (main) flyouts should use the standard `euiFlyoutHeader` structure, which we already render in our layouts.
The title generated by the new flyout system is intended only for child flyouts and is rendered inside the `euiFlyoutMenu` markup.
In a future implementation we should ensure that this auto-generated menu title is not shown for parent flyouts, as it does not match the expected design.